tests/openvino/test_quantization.py

```diff
@@ -0,0 +1,63 @@
+#  Copyright 2021 The HuggingFace Team. All rights reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import tempfile
+import unittest
+from functools import partial
+
+from transformers import AutoModelForSequenceClassification, AutoTokenizer
+
+from optimum.intel.openvino.modeling import OVModelForSequenceClassification
+from optimum.intel.openvino.nncf_config import DEFAULT_QUANTIZATION_CONFIG
+from optimum.intel.openvino.quantization import OVQuantizer
+from parameterized import parameterized
+
+
+class OVQuantizerTest(unittest.TestCase):
+    SUPPORTED_ARCHITECTURES_WITH_EXPECTED_QUANTIZED_MATMULS = (
+        ("distilbert-base-uncased-finetuned-sst-2-english", 82),
+    )
+
+    @parameterized.expand(SUPPORTED_ARCHITECTURES_WITH_EXPECTED_QUANTIZED_MATMULS)
+    def test_static_quantization(self, model_name, expected_fake_quantize):
```

Comment:
Could we have a test that verifies that outputs are as expected? I would like to have at least 1 sanity check that accuracy is not impacted on an update.

Reply:
Sure, I will add the test